pkg/sources/chunker_test.go

```diff
@@ -7,8 +7,8 @@ import (
 	"testing"
 	"testing/iotest"
 
-	diskbufferreader "github.com/trufflesecurity/disk-buffer-reader"
 	"github.com/stretchr/testify/assert"
+	diskbufferreader "github.com/trufflesecurity/disk-buffer-reader"
```

Comment:
Note: not sure how/why this got updated, but i'm going to leave it since its just a import reorder.